var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -968,6 +973,8 @@ def cmake_args(self):
                     self.define("CP2K_WITH_GPU", gpu_ver),
                 ]
 
+        # we need to add the quip package to spack.
+
```

Comment:
I think this would be better off as an issue (in Spack), instead of a comment here? I'd also remove the related code, instead of commenting it out. I don't know what the best practice is in Spack.

Reply:
Agreed. I removed the comment and will open an issue (eventually).